adders/tailwindcss/config/adder.ts

```diff
@@ -72,10 +72,18 @@ export const adder = defineAdderConfig({
             name: () => "src/app.css",
             contentType: "css",
             content: ({ ast, addAtRule }) => {
-                const atRules = ["utilities", "components", "base"];
-                for (const name of atRules) {
-                    addAtRule(ast, "tailwind", name);
+                const imports = ['"tailwindcss/utilities"', '"tailwindcss/components"', '"tailwindcss/base"'];
+                const firstNode = ast.first;
```

Comment:
I think something like `originalFirst` might be clearer when used later. E.g. `firstNode !== ast.first` vs `originalFirst !== ast.first`